biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.java

```diff
@@ -222,6 +228,22 @@ private String _write_the_first_line(S sequence) {
 		 */
 	}
 
+	/**
+	 * Write the original LOCUS line.
+	 *
+	 * @param sequence
+	 */
+	private String _write_original_first_line(S sequence) {
```

Comment:
Nitpick: could you keep the method name as per usual camel case convention?

Reply:
Sure happy to change, I would normally but I was just following the convention already used in the class
e.g. _write_single_line, _write_multi_line etc

Reply:
Oh I didn't realise that was the style already for that file. Well I guess we can live with that a bit longer...